### PR TITLE
Separate questionnaires

### PR DESCRIPTION
**Pull request by saulakravitz**
1) Put Questionnaires in their own grouping in artifacts.
2) Align state, territory, country fields with VS from VRCL
3) Align race/ethnicity with yes/no + literal approach (rather than having the mom code her race/ethnicity).
4) Align QuestionnaireResponses with FHIR example content

### CODE REVIEW

File: input/fsh/instances/QuestionnaireResponse-patients-fetal-death-carmen-lee.fsh [hunk 1]
```diff
@@ -68,18 +68,12 @@ Usage: #example
   * answer.valueCoding = $HL7_EducationLevelCS#SEC "Some secondary or high school education"
 * item[+]
   * linkId = "mother-ethnicity"
-  * text = "Are you Spanish/Hispanic/Latina? If not Spanish/Hispanic/Latina, check the “No” box. If Spanish/Hispanic/Latina, check the appropriate box."
-  * item
-    * linkId = "mother-shl"
-    * text = "Are you Spanish/Hispanic/Latina"
-    * answer.valueCoding = urn:oid:2.16.840.1.113883.6.238#2135-2 "Hispanic or Latino"
+  * answer.valueBoolean = true
```
Comment (rpassas) [inline input/fsh/instances/QuestionnaireResponse-patients-fetal-death-carmen-lee.fsh:71]: The questionnaire says: "9. Are you Spanish/Hispanic/Latina? (if so, please answer 9a-9d)"

Entries a-d are not mandatory (0..1), but I imagine the ethnicity type should be addressed in a-d in the example unless we want to indicate "unknown" in the FHIR example. The [Observation - Input Race and Ethnicity Vital Records](https://build.fhir.org/ig/HL7/vr-common-library//StructureDefinition-input-race-and-ethnicity-vr.html) FHIR example indicates Puerto Rican, Cuban, and Mexican for Carmen Teresa Lee.

Also, is there a reason the text should be removed? I think it gives good context in the example.

File: input/fsh/questionnaires/Questionnaire-mothers-live-birth.fsh [hunk 2]
```diff
@@ -198,43 +210,146 @@ Usage: #example
...
+    * type = #boolean
+    * repeats = false
+  * item[+]
+    * linkId = "mother-race-filipino"
+    * text = "Filipino"
+    * type = #boolean
+    * repeats = false
+  * item[+]
+    * linkId = "mother-race-japanese"
+    * text = "Chinese"
```
Comment (ptran265) [inline input/fsh/questionnaires/Questionnaire-mothers-live-birth.fsh]: this should say Japanese

File: input/fsh/questionnaires/Questionnaire-mothers-live-birth.fsh [hunk 3]
```diff
@@ -444,43 +563,147 @@ Usage: #example
...
-      * question = "father-shl"
-      * operator = #=
-      * answerCoding = urn:oid:2.16.840.1.113883.6.238#2135-2
+    * linkId = "father-ethnicity-mexican"
+    * text = "Is the father Mexican, Mexican-American?"
+    * type = #boolean
+    * repeats = false
+  * item[+]
+    * linkId = "father-ethnicity-puerto-rican"
+    * text = "Is the father you Puerto Rican?"
```
Comment (ptran265) [inline input/fsh/questionnaires/Questionnaire-mothers-live-birth.fsh]: typo "you"

File: input/fsh/questionnaires/Questionnaire-mothers-live-birth.fsh [hunk 4]
```diff
@@ -444,43 +563,147 @@ Usage: #example
...
+    * type = #boolean
+    * repeats = false
+  * item[+]
+    * linkId = "father-ethnicity-cuban"
+    * text = "Is the father you Cuban?"
```
Comment (ptran265) [inline input/fsh/questionnaires/Questionnaire-mothers-live-birth.fsh]: typo

File: input/fsh/questionnaires/Questionnaire-mothers-live-birth.fsh [hunk 5]
```diff
@@ -444,43 +563,147 @@ Usage: #example
...
+    * linkId = "father-ethnicity-other"
+    * text = "Is the father of other Spanish/Hispanic/Latina (e.g., Spaniard, Salvadoran, Dominican, Columbian) (specify)."
+    * type = #string
     * repeats = false
-    * answerValueSet = "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.114222.4.11.877"
 * item[+]
   * linkId = "father-race"
   * prefix = "24"
-  * text = "What is the father's race? (Please check one or more races to indicate what he considers himself to be)."
+  * text = "What is the father's race? (Please check one or more races to indicate what what he considers himself to be)."
```
Comment (ptran265) [inline input/fsh/questionnaires/Questionnaire-mothers-live-birth.fsh]: typo "what"

File: input/fsh/questionnaires/Questionnaire-mothers-live-birth.fsh [hunk 6]
```diff
@@ -444,43 +563,147 @@ Usage: #example
...
+    * type = #boolean
+    * repeats = false
+  * item[+]
+    * linkId = "father-race-filipino"
+    * text = "Filipino"
+    * type = #boolean
+    * repeats = false
+  * item[+]
+    * linkId = "father-race-japanese"
+    * text = "Chinese"
```
Comment (ptran265) [inline input/fsh/questionnaires/Questionnaire-mothers-live-birth.fsh]: should be Japanese

File: input/fsh/questionnaires/Questionnaire-patients-fetal-death.fsh [hunk 7]
```diff
@@ -86,15 +86,17 @@ Usage: #example
   * item[+]
     * linkId = "mother-state"
     * text = "State (or U.S. Territory, Canadian Province)"
-    * type = #string
+    * type = #choice
+    * answerValueSet =   Canonical(ValueSetUSStatesVitalRecords)  //    "http://phinvads.cdc.gov/fhir/ValueSet/2.16.840.1.114222.4.11.888"
```
Comment (ptran265) [inline input/fsh/questionnaires/Questionnaire-patients-fetal-death.fsh]: should this be Canonical(ValueSetStatesTerritoriesAndProvincesVitalRecords)?

File: input/fsh/questionnaires/Questionnaire-patients-fetal-death.fsh [hunk 8]
```diff
@@ -182,15 +186,21 @@ Usage: #example
   * item[0]
     * linkId = "mother-birthplace-state"
     * text = "State"
-    * type = #string
+    * type = #choice
+    * repeats = false
+    * answerValueSet = Canonical(ValueSetStatesTerritoriesAndProvincesVitalRecords)  // Should be just states
```
Comment (ptran265) [inline input/fsh/questionnaires/Questionnaire-patients-fetal-death.fsh]: should this be Canonical(ValueSetUSStatesVitalRecords)?

File: input/fsh/questionnaires/Questionnaire-patients-fetal-death.fsh [hunk 9]
```diff
@@ -182,15 +186,21 @@ Usage: #example
...
   * item[+]
     * linkId = "mother-birthplace-territory"
     * text = "or U.S. territory, i.e., Puerto Rico, U.S. Virgin Islands, Guam, American Samoa or Norther Marianas"
-    * type = #string
+    * repeats = false
+    * type = #choice
+    * answerValueSet = Canonical(ValueSetStatesTerritoriesAndProvincesVitalRecords) // SHould be just states
```
Comment (ptran265) [inline input/fsh/questionnaires/Questionnaire-patients-fetal-death.fsh]: and this Canonical(ValueSetUSTerritoriesVitalRecords)?

File: input/fsh/questionnaires/Questionnaire-patients-fetal-death.fsh [hunk 10]
```diff
@@ -202,43 +212,147 @@ Usage: #example
...
+    * type = #boolean
+    * repeats = false
+  * item[+]
+    * linkId = "mother-race-filipino"
+    * text = "Filipino"
+    * type = #boolean
+    * repeats = false
+  * item[+]
+    * linkId = "mother-race-japanese"
+    * text = "Chinese"
```
Comment (ptran265) [inline input/fsh/questionnaires/Questionnaire-patients-fetal-death.fsh]: Japanese

**Review by ptran265 — CHANGES_REQUESTED** [review]

File: input/fsh/instances/QuestionnaireResponse-patients-fetal-death-carmen-lee.fsh (hunk 1 above)
Comment (saulakravitz) [reply input/fsh/instances/QuestionnaireResponse-patients-fetal-death-carmen-lee.fsh:71]: Good point....Adding detailed ethnicity to the example questionnaire to line up with the example in FHIR.

Not sure what you mean by "text should be removed"?

**Pull request merged**